oracle/client/client.go

```diff
@@ -89,6 +89,8 @@ func NewOracleClient(
 		GRPCEndpoint:        grpcEndpoint,
 	}
 
+	authtypes.RegisterInterfaces(oracleClient.Encoding.InterfaceRegistry)
```

Comment:
Could this line be somewhere in initialization / app.go in the Ojo repo?
Especially if we use this encoding for multiple clients other than PF